src/cpp/fastdds/publisher/DataWriterImpl.hpp

```diff
@@ -586,9 +586,16 @@ class DataWriterImpl : protected rtps::IReaderDataFilter
             const DataWriterQos& from,
             bool is_default);
 
+    static ReturnCode_t check_qos_including_resource_limits(
```

Comment:
New methods should be documented.

Reply:
188518d2f Refs #14958: New DatawriterImpl methods are now documented.